src/common/class-syntax.k

```diff
@@ -20,7 +21,7 @@ syntax ClassMod ::= Public
                   | StrictFP
 syntax AnnoClassMod ::= Anno | ClassMod
 
-syntax Super ::= "extends" ClassType                    [klabel('SuperDec)]
+syntax Super ::= "extends" JavaClassType                    [klabel('SuperDec)]
 syntax Interfaces ::= "implements" InterfaceTypeList    [klabel('ImplementsDec)]
```

Comment:
We really, really like things which are properly aligned. Please add the attributes at the end in alignment with the rest of the productions in the block. And could make this a habit. It helps readability greatly.